packages/dev-utils/src/constants.js

```diff
@@ -47,27 +67,22 @@ module.exports = {
                 { name: 'verifyingContract', type: 'address' },
             ],
         },
-        primaryType: 'AZTEC_NOTE_SIGNATURE',
-    },
-    ACE_DOMAIN_PARAMS: {
-        name: 'AZTEC_CRYPTOGRAPHY_ENGINE',
-        version: '1',
+        primaryType: 'AZTECNoteSignature',
     },
-    ACE_NOTE_SIGNATURE: {
+    AZTEC_NOTE_SIGNATURE_V2: {
```

Comment:
we should get rid of this as it's obsolete (to be rolled into a separate issue, like the above comment)

Reply:
It seems you commented on `AZTEC_NOTE_SIGNATURE_V2`. Shouldn't `AZTEC_NOTE_SIGNATURE` be the one to nuke out?

Reply:
Created an issue for this: https://github.com/AztecProtocol/AZTEC/issues/74